src/content/en/updates/2011/11/Quota-Management-API-Fast-Facts.md

```diff
@@ -8,46 +8,45 @@ book_path: /web/updates/_book.yaml
-    * If app tries to make a modification which will result in exceeding `TEMPORARY` storage quota (20% of pool), an error will be thrown.
-    * Each application can query how much data is stored or how much more space is available for the app by calling `queryUsageAndQuota()` method of Quota API.
-    * Requesting more quota against `TEMPORARY` storage doesn’t do anything.
-    * Requesting quota for `TEMPORARY` storage using `webkitRequestFileSystem()` doesn’t actually allocate / change quota.
+    * When `TEMPORARY` storage quota is exceeded, _all the data (incl. AppCache, IndexedDB, WebSQL, File System API) stored for oldest used origin gets deleted_.<br>[Note: since each app can only use up to 20% of the pool, this won’t happen very frequently unless the user has more than 5 active offline apps.]
+    * If an app tries to make a modification which will result in exceeding `TEMPORARY` storage quota (20% of pool), an error will be thrown.
+    * Each application can query how much data is stored or how much more space is available for the app by calling the `queryUsageAndQuota()` method of Quota API.
 
+          webkitStorageInfo.queryUsageAndQuota(
+              webkitStorageInfo.TEMPORARY,   // or PERSISTENT
+              usageCallback,
+              errorCallback);
 
-    webkitStorageInfo.queryUsageAndQuota(
-      webkitStorageInfo.TEMPORARY,   // or PERSISTENT
-      usageCallback,
-      errorCallback);
-    
+    * Requesting more quota against `TEMPORARY` storage doesn’t do anything.
+    * Requesting quota for `TEMPORARY` storage using `webkitRequestFileSystem()` doesn’t actually allocate / change quota.
 
 * For `PERSISTENT` storage, its default quota is 0 and it needs to be explicitly requested per application using `requestQuota()` of Quota API.
     * The allocated space can be used only by File System API (for `PERSISTENT` type filesystem) and there’s no such thing like `PERSISTENT` storage on _IndexedDB, WebSQL DB or AppCache_ (yet).
 
 
-    webkitStorageInfo.requestQuota(
-      webkitStorageInfo.PERSISTENT
-      newQuotaInBytes,
-      quotaCallback,
-      errorCallback);
-    
+          webkitStorageInfo.requestQuota(
+              webkitStorageInfo.PERSISTENT
+              newQuotaInBytes,
+              quotaCallback,
+              errorCallback);
+
 
-* `unlimitedStorage` on manifest.json of Chrome Web App has been brought as a temporary solution for apps to work without Quota API. So, there’s no guarantee that Chrome will support this feature forever.
+* Setting `unlimitedStorage` in the `manifest.json` of Chrome Web Apps has been suggested as a temporary solution for apps to work without Quota API. There is no guarantee that Chrome will continue supporting this feature indefinitely.
```

Comment:
'continue supporting this feature indefinitely' is slightly redundant. Pick one of:

* continue supporting this feature
* support this feature indefinitely